tests/unit_tests/regression/test_aerodyn_inflow.cpp

```diff
@@ -0,0 +1,40 @@
+#include <gtest/gtest.h>
+
+#include "test_utilities.hpp"
+
+#include "src/vendor/dylib/dylib.hpp"
+
+namespace openturbine::tests {
+
+TEST(AerodynInflowTest, ADI_C_PreInit) {
+    // Use dylib to load the dynamic library and get access to the aerodyn inflow c binding functions
+    auto path = std::string{"libaerodyn_inflow_c_binding."};
+#ifdef __APPLE__
+    path += "dylib";
+#elif __linux__
+    path += "so";
+#else if _WIN32 || _WIN64
+    path += "dll";
+#endif
+    const util::dylib lib(path, util::dylib::no_filename_decorations);
+    auto ADI_C_PreInit = lib.get_function<void(int*, int*, int*, int*, char*)>("ADI_C_PreInit");
```

Comment:
If our coupling is through a shared library, is this not just testing that a shared library file exists with that function in it?  If so, we could just make our own for simple testing purposes and then require the full aerodyn one only when we actually need its functionality.

Reply:
We are testing the simplest method from the external library here, this will grow in the next PR(s) as we create interface in openturbine for calling the methods we need from aerodyn inflow library.

Reply:
Since we're building aerodyn ourselves and disabling the unit tests when we don't, this should be fine